src/main/java/team/gdsc/code2cv/global/controller/error/ApiExceptionControllerAdvice.java

```diff
@@ -1,70 +1,94 @@
 package team.gdsc.code2cv.global.controller.error;
 
-import io.jsonwebtoken.JwtException;
-import lombok.extern.slf4j.Slf4j;
+import java.util.List;
+
 import org.springframework.http.HttpStatus;
 import org.springframework.validation.BindingResult;
 import org.springframework.validation.FieldError;
 import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
-import team.gdsc.code2cv.core.exception.ResourceNotFoundException;
 
-import java.util.List;
+import io.jsonwebtoken.JwtException;
+import lombok.extern.slf4j.Slf4j;
+import team.gdsc.code2cv.core.exception.ExternalServerCommunicationException;
+import team.gdsc.code2cv.core.exception.ResourceNotFoundException;
+import team.gdsc.code2cv.core.exception.TokenExpiredException;
 
 @RestControllerAdvice
 @Slf4j
 public class ApiExceptionControllerAdvice {
-    @ExceptionHandler
-    @ResponseStatus(HttpStatus.BAD_REQUEST)
-    public ErrorResponse illegalArgumentExceptionHandler(IllegalArgumentException e){
-        log.debug("IllegalArgumentException : {}", e.getMessage());
-        return ErrorResponse.builder()
-                .debugMessage(e.getMessage())
-                .code("IllegalArgumentException")
-                .build();
-    }
+	@ExceptionHandler
+	@ResponseStatus(HttpStatus.BAD_REQUEST)
+	public ErrorResponse illegalArgumentExceptionHandler(IllegalArgumentException error) {
+		log.debug("IllegalArgumentException : {}", error.getMessage());
+		return ErrorResponse.builder()
+			.debugMessage(error.getMessage())
+			.code("IllegalArgumentException")
+			.build();
+	}
```

Comment:
```suggestion
	public ErrorResponse illegalArgumentExceptionHandler(IllegalArgumentException ex) {
		log.debug("IllegalArgumentException : {}", ex.getMessage());
		return ErrorResponse.builder()
			.debugMessage(ex.getMessage())
			.code("IllegalArgumentException")
			.build();
	}
```

아래는 다 parameter 이름이 ex인데 여기만 error네요. 굳이 다를 이유가 없다면 ex로 통일하는게 좋을 것 같습니다.

Reply:
넵 좋습니다!